### ISSUE TITLE
Allow to fail CI

### ISSUE THREAD

**Issue by dbartholomae** [smeijer/unimported#73]
This sounds like a great tool to enforce removing superfluous code once it is no longer used. To do this, a `--ci` flag would be great that fails if any unimported files are present. Bonus points if it also fails for any ignore that is not needed.

**Comment by smeijer** [comment]
Hi! Thanks for the suggestion. 

Unimported already exits with a non zero code when there is something to report. So you can already use it in ci for that purpose.

We currently don't validate the config file. Although it would be a nice addition, I feel like it could get quite complex to add that, as the config file also supports patterns here and there.

Do you experience any issues with running in ci, that lead to opening this feature request?

**Comment by dbartholomae** [comment]
Just tried it - functionality is fine. I think I'm just used to having some red message at the end of a command that fails :D
Maybe a heading on CI usage in README might help. Something like:

> ### CI Usage
>
> You can drop in `npx unresolved` into your CI. It will fail if it finds any unimported files that are not explicitly set up in the unresolved config file.

**Comment by smeijer** [comment]
Yeah, that makes sense. Are you willing and able to submit a pull request with that suggested change?

**Comment by dbartholomae** [comment]
Here you are: #75
